models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1796,7 +1796,8 @@ FROM
     ('giv-giv', 'ethereum', 'GIV', 0x900db999074d9277c5da2a43f252d74366230da0, 18),
     ('zro-layerzero', 'ethereum', 'ZRO', 0x6985884c4392d348587b19cb9eaaf157f13271cd, 18),
     ('quartz-sandclock', 'ethereum', 'QUARTZ', 0xba8a621b4a54e61c442f5ec623687e2a942225ef, 18),
-    ('wcomai-wrapped-commune-ai', 'ethereum', 'WCOMAI', 0xc78b628b060258300218740b1a7a5b3c82b3bd9f, 18)
+    ('wcomai-wrapped-commune-ai', 'ethereum', 'WCOMAI', 0xc78b628b060258300218740b1a7a5b3c82b3bd9f, 18),
+    ('nmt-netmind-token', 'ethereum', 'NMT', 0x03aa6298f1370642642415edc0db8b957783e8d6, 18)
```

Comment:
This token may have been already defined here:
https://github.com/duneanalytics/spellbook/blob/4a480d57025551aa2051464b83e2f182e1cd8ddb/models/prices/ethereum/prices_ethereum_tokens.sql#L1753